frontend/src/components/App.vue

```diff
@@ -1,21 +1,44 @@
 <template>
   <div id="app" class="container">
-    <div class="jumbotron">
-      <div class="project-title__wrapper">
-        <div class="media-left media-middle">
-          <img class="logo-image" src="../assets/logo.png">
-        </div>
-        <div class="media-body">
-          <h1 class="project-name">Outrigger Dashboard</h1>
-        </div>
+
+    <nav class="navbar navbar-light bg-light navbar-expand-lg fixed-top">
+      <a class="navbar-brand" href="#"><img src="../assets/logo.png" height="40" width="40" alt="Outrigger Logo" /></a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarNavDropdown">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item"><a class="nav-link" href="http://outrigger.sh" title="Outrigger Homepage">What is Outrigger?</a></li>
+          <li class="nav-item"><a class="nav-link" href="http://docs.outrigger.sh" title="Technical Documentation">Documentation</a></li>
+          <li class="nav-item"><a class="nav-link" href="https://github.com/search?o=desc&q=topic%3Aoutrigger+org%3Aphase2&s=updated&type=Repositories" title="Review all the Outrigger repositories on Github"><i class="fa fa-github" aria-hidden="true"></i>&thinsp;Contribute</a></li>
+          <li class="nav-item"><a class="nav-link" href="http://slack.outrigger.sh/" title="Join us on Slack!"><i class="fa fa-slack" aria-hidden="true"></i>&thinsp;Slack</a></li>
+        </ul>
+        <ul class="navbar-nav ml-auto">
+          <li role="separator" class="dropdown-divider"></li>
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" id="supportDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              File an Issue
+            </a>
+            <div class="dropdown-menu dropdown-menu-right" aria-labelledby="supportDropdown">
+              <a class="dropdown-item" href="https://github.com/phase2/outrigger-dashboard/issues">Dashboard issues</a>
+              <a class="dropdown-item" href="https://github.com/phase2/outrigger-docs/issues">Documentation issues</a>
+              <a class="dropdown-item" href="https://github.com/phase2/rig/issues">Outrigger CLI (rig) issues</a>
+            </div>
+          </li>
+        </ul>
       </div>
-    </div>
-    <div class="container">
+    </nav>
+
+    <header class="jumbotron">
+      <h1 class="display-3 mt-5">Outrigger Dashboard</h1>
```

Comment:
`mt-5` stands for `margin top 5`, where 5 is multiplied by some kind of predefined spacer and unit of measurement. display-3 addresses some sizing issues after I pulled out the extra markup.